x-pack/plugins/security_solution/public/timelines/components/timeline/body/events/stateful_event.tsx

```diff
@@ -127,7 +126,7 @@ const StatefulEventComponent: React.FC<Props> = ({
     if (timelineId === TimelineId.active) {
       activeTimeline.toggleExpandedEvent({ eventId, indexName, loading: false });
     }
-  }, [dispatch, event._id, event._index, timelineId]);
+  }, [dispatch, timelineId, event]);
```

Comment:
are we sure that we want to always re-render that whenever any of the `event` props change?